.github/workflows/package.yaml

```diff
@@ -136,8 +136,11 @@ jobs:
         source: './dist/rancher-desktop-canary.zip'
         dest: 's3://rancher-desktop-assets-for-obs/rancher-desktop-canary.zip'
     - name: Trigger OBS services
-      uses: distributhor/workflow-webhook@v2
       if: steps.has_s3.outcome == 'success'
+      run: |
+        curl -X POST \
+          -H "Authorization: Token ${webhook_secret}" \
+          "${webhook_url}"
       env:
         webhook_type: 'form-urlencoded'
```

Comment:
The webhook_type is no longer being used. It can be removed.

You can also skip the env part and just use the secrets directly. The env was needed for the container that was being used before.

Reply:
I removed the webhook_type env var. However I believe the other env vars need to be kept: https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow